server/blob/blob-deduplicating/pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>testing-base</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.typesafe.play</groupId>
+            <artifactId>play-json_${scala.base}</artifactId>
+            <version>2.8.1</version>
```

Comment:
We should use the same version than in `mailbox/event/json` and JMAP RFC-8621 as far as I understand